cmd/config/testdata/config

```diff
@@ -1,6 +1,9 @@
-api-url: api.screwdriver.com
-store-url: store.screwdriver.com
-token: sd-token
-launcher:
-  version: 1.0.0
-  image: screwdrivercd/launcher
+configs:
+  default:
+    api-url: api.screwdriver.com
+    store-url: store.screwdriver.com
+    token: sd-token
+    launcher:
+      version: 1.0.0
+      image: screwdrivercd/launcher
+current: default
```

Comment:
This is a workaround to avoid failing tests.

Tests for `sd-local config view` will be modified by another PR.